### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by kalj**

### CODE REVIEW

**Comment by S-Dafarra** [comment]
Oh, thanks for catching! Instead of simply removing the old version, can you deprecate the old version (using the ``c++14`` [``deprecated``](https://en.cppreference.com/w/cpp/language/attributes/deprecated) keyword) instead?

**Comment by kalj** [comment]
> Oh, thanks for catching! Instead of simply removing the old version, can you deprecate the old version (using the `c++14` [`deprecated`](https://en.cppreference.com/w/cpp/language/attributes/deprecated) attribute) instead?

Ah, I see. Sure.

File: include/OsqpEigen/Settings.hpp [hunk 1]
```diff
@@ -89,7 +89,13 @@ namespace OsqpEigen
...
          * @param maxIteration max number of iteration
          */
-        void setMaxIteraction(const int maxIteration);
+        [[deprecated("Use setMaxIteration(int) instead.")]] void setMaxIteraction(const int maxIteration);
+
+        /**
+         * Set the max number of iterations.
+         * @param maxIteration max number of iteration
+         */
+		void setMaxIteration(const int maxIteration);
```
Comment (traversaro) [inline include/OsqpEigen/Settings.hpp:98]: Thanks @kalj ! Can you keep using spaces instead of tabs, as done in the rest of the file?

Comment (kalj) [reply include/OsqpEigen/Settings.hpp:98]: Oh, crap. Absolutely.

**Review by S-Dafarra — APPROVED** [review]

**Comment by S-Dafarra** [comment]
We can wait for the CI to finish and, if @GiulioRomualdi agrees, we can merge.

**Pull request merged**